Accelerate gradient data for constant and Aitken underrelaxation (#1373)
precice/precice: docs/changelog/1373.md

> - Added support for gradient data acceleration for constant underrelaxation and Aitken acceleration.

Reviewer: ```suggestion
- Added gradient data to constant and Aitken underrelaxation.
```
